Makefile

```diff
@@ -1,5 +1,6 @@
 # Define the target to run if make is called with no arguments.
 default: check
+UNAME_S := $(shell uname -s)
```

Comment:
This variable implies the `OS`, so let's call it so. Also a good practice is to lowercase everything `shell uname -s | tr '[A-Z]' '[a-z]'`